TestTools/StreamChatTestTools/Mocks/StreamChat/VoiceRecording/MockAudioPlayer.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+public class MockAudioPlayer: AudioPlaying {
+
+    public private(set) var subscribeWasCalledWithSubscriber: AudioPlayingDelegate?
+
+    public private(set) var loadAssetWasCalledWithURL: URL?
+
+    public private(set) var clearUpQueueWasCalled = false
+
+    public private(set) var playWasCalled = false
+
+    public private(set) var pauseWasCalled = false
+
+    public private(set) var stopWasCalled = false
```

Comment:
nit: Usually it is a best practice to use a `callCount: Int` instead of a boolean, to make sure we called the method only 1 time, since calling multiple times can introduce problems, and with a boolean we are not able to catch this 👍

Reply:
That's a good suggestion. I have added a note in the doc we have for the remaining tasks and I can tackle it more easily once all changes are in the feature branch.